src/cbc_syslog/util/carbon_black_cloud.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, sources):
                 "server_url": "",
                 "audit_logs_enabled": True,
                 "alerts_enabled": True,
-                "alert_criteria": {
+                "alert_rules": {
                     "type": [
```

Comment:
how sensitive is this to new alert types?  e.g. if someone just asks for "all alerts" will the receive new types without having to make updates? Would be nice if the get the base fields automatically.

Reply:
This supports any and all properties as long as they are a list of values. At the moment we have to manually add support for any fields which are not a list of values given the SDK `add_criteria` doesn't support non array values.

This will enable a customer to specify exaclty which alert types they want or export all if no types or criteria are specified